python/cuml/benchmark/run_benchmarks.py

```diff
@@ -52,6 +52,19 @@ def extract_param_overrides(params_to_sweep):
 
 if __name__ == '__main__':
     import argparse
+    import sys
+
+    class PrintAlgorithms(argparse.Action):
+        def __call__(self, parse, namespace, values, option_string=None):
+            for algo in algorithms.all_algorithms():
```

Comment:
These don't really need special argparse actions. You can just add them as flags to argparse then do:
```
if args.print_datasets:
 for dataset ....
   print(dataset)
```

then the two new classes aren't needed.

Reply:
Done.